sdk/go/common/resource/config/value.go

```diff
@@ -240,6 +240,53 @@ func isSecureValue(v interface{}) (bool, string) {
 	return false, ""
 }
 
+func encryptObject(v interface{}, decrypter Decrypter, encrypter Encrypter) (interface{}, error) {
```

Comment:
Consider renaming this to `reencryptObject` since it's doing more than just _encrypting_ -- it's _decrypting_ and the _re-encrypting_.

```suggestion
func reencryptObject(v interface{}, decrypter Decrypter, encrypter Encrypter) (interface{}, error) {
```

Reply:
👍